scripts/apps/authoring/authoring/directives/AuthoringEmbeddedDirective.js

```diff
@@ -1,14 +1,35 @@
 import * as helpers from 'apps/authoring/authoring/helpers';
 
-AuthoringEmbeddedDirective.$inject = ['api', 'notify', 'gettext', '$filter', 'config'];
-export function AuthoringEmbeddedDirective(api, notify, gettext, $filter, config) {
+AuthoringEmbeddedDirective.$inject = ['api', 'notify', 'gettext', '$filter', 'config', 'deployConfig'];
+export function AuthoringEmbeddedDirective(api, notify, gettext, $filter, config, deployConfig) {
     return {
         templateUrl: 'scripts/apps/authoring/views/authoring.html',
         scope: {
             item: '=',
             action: '='
         },
         link: function(scope) {
+            function overrideEdnote(template) {
+                /* Override Editor note with given template or default one
+                 *
+                 * @param {string} template - template to use (set in backend)
+                 */
+                let date = $filter('formatLocalDateTimeString')(scope.item.versioncreated, config.view.dateformat +
+                    ' ' + config.view.timeformat);
+
+                if (template == null) {
+                    // no template specified in backend, we use default one
+                    let slugline = scope.item.slugline ? '"' + scope.item.slugline + '" ' : '';
+
+                    scope.item.ednote = gettext('In the story ') + slugline + gettext('sent at: ') + date +
```

Comment:
I know it was like that before, but imo would be better to use $interpolate so there is single string for translation:
```
scope.item.ednote = $interpolate(gettext('In the story {{ slugline }} sent at: {{ date }}\r\n\r\nThis is corrected repeat.'))({slugline, date});
```

Reply:
yes it was like that before, but you are right, I will change it.